src/shared/Navbar.tsx

```diff
@@ -28,6 +34,17 @@ function Navbar() {
     setAnchorEl(null);
   };
 
+  const handleExport = async () => {
+    const zip = new JSZip();
+    zip.file("families.csv", await FamilyAPI.exportFamilies());
+    zip.file("fields.csv", await FieldsAPI.exportFields());
+    zip.file("sessions.csv", await SessionAPI.exportSessions());
+    zip.file("students.csv", await StudentAPI.exportStudents());
```

Comment:
i think we're missing enrolments and classes here? :0

Reply:
howd i miss that...